seatunnel-api/src/main/java/org/apache/seatunnel/api/table/type/SeaTunnelRow.java

```diff
@@ -50,7 +50,16 @@ public void setTableId(String tableId) {
         this.tableId = tableId;
     }
 
+    /**
+     * The method will be removed in the future, please use {@link #setKind(RowKind)} instanced of
+     * it.
+     */
+    @Deprecated
     public void setRowKind(RowKind kind) {
+        setKind(kind);
+    }
```

Comment:
Consider it more carefully, why not change `kind` field to `rowKind`? cc @hailin0

Reply:
Do you guys have any suggestions?
@Carl-Zhou-CN @TyrantLucifer

Reply:
I think this is a good approach if it achieves the same effect; modifying fields has less impact and avoids the issue of deprecated methods.